src/components/CallView/shared/VideoVue.vue

```diff
@@ -22,8 +22,12 @@
 	<div v-show="!placeholderForPromoted || sharedData.promoted"
 		:id="(placeholderForPromoted ? 'placeholder-' : '') + 'container_' + peerId + '_video_incoming'"
 		ref="videoContainer"
-		class="videoContainer"
-		:class="containerClass"
+		class="video-container"
+		:class="[containerClass, {
+			speaking: isSpeaking,
+			hover: mouseover && !unSelectable,
```

Comment:
```suggestion
			hover: mouseover && !unSelectable  && !isBig,
```
Same here, Big is not an interactable element